client/src/components/LayoutComponents/Form/index.jsx

```diff
@@ -0,0 +1,188 @@
+			const newCheckedArry = this.state.questions.map((question) => {
+				if (index === question.id) {
+					question.checked = !question.checked;
+				}
+				return question;
+			});
+
+			return this.setState({ questions: newCheckedArry });
+		}
+		this.setState({ [name]: value });
+	};
+	handleSelectNewQuestionType = (event) => {
+		this.setState({ newQuestionType: event.target.value });
+	};
+	handleAddQuestionClick = (event) => {
+		event.preventDefault();
+		if (
+			this.state.newQuestionType === '' ||
+			this.state.newQuestionText === ''
+		) {
+			return this.setState({
+				typeError:
+					'Please insert the quesstion & choose the type of the Question!!',
+			});
+		}
+		this.setState({ typeError: '' });
+		axios
+			.post('/api/dashboard/form', {
+				content: this.state.newQuestionText,
+				type: this.state.newQuestionType,
+			})
+			.then(({ data }) => {
+				if (!data.message) return this.setState({ error: data.message });
+				const newId = this.state.questions.length;
+				let json = {
+					id: newId,
+					context: this.state.newQuestionText,
+					checked: false,
+				};
+				return this.setState((prevState) => {
+					return { questions: [...prevState.questions, json] };
+				});
+			})
+			.catch((err) =>
+				this.setState({ error: 'There was an error adding your question' }),
+			);
+	};
+
+	handleSubmit = (event) => {
+		event.preventDefault();
+		this.setState({ submitDialogOpen: true });
+	};
+	handleClose = () => {
+		this.setState({ submitDialogOpen: false });
+	};
+	render() {
+		return (
+			<Fragment>
+				<Grid>
+					{this.state.loading ? (
+						<h1>Please wait its loading </h1>
+					) : this.state.error ? (
+						<Alert severity="error">{this.state.error}</Alert>
+					) : (
+						this.state.questions.map((question, index) => {
+							return (
+								<FormControlLabel
+									control={
+										<Checkbox
+											checked={question.checked}
+											onChange={(event) => this.handleChange(index, event)}
+											name={question.context}
+											color="primary"
+										/>
+									}
+									label={question.context}
+									key={index}
+								/>
+							);
+						})
+					)}
+				</Grid>
+				<Grid>
+					<TextField
+						id="outlined-basic"
+						label="write a question ..."
+						variant="outlined"
+						name="newQuestionText"
+						onChange={(event) => this.handleChange(null, event)}
+					/>
+				</Grid>
+				{this.state.typeError ? (
+					<Alert severity="warning">{this.state.typeError}</Alert>
+				) : null}
+				<Grid>
+					<FormControl>
+						<InputLabel id="select-label">Type</InputLabel>
+						<Select
+							labelId="select-label"
+							id="demo-simple-select"
+							label="Type"
+							value={this.state.newQuestionType}
+							onChange={this.handleSelectNewQuestionType}
+						>
+							<MenuItem value="Rate">Rate</MenuItem>
+							<MenuItem value="Content">Content</MenuItem>
+							<MenuItem value="Mullti">Mullti</MenuItem>
```

Comment:
Backend PR doesn't matter there will be another branch to do all of that

Reply:
Change types based on DB!! 
I can't understand your way to fix things and comments! check my comments again and do or discuss them

Reply:
listen, this branch the only goal here is a backend for the front end it will totally change its just a test from front end to backend btw did Hussien deleted the dummy data for the questions?